pyproject.toml

```
@@ -29,7 +29,7 @@ classifiers = [
 ]
 dependencies = [
     "certifi",
-    "httpcore>=0.18.0,<0.19.0",
+    "httpcore",
```

Since `httpcore 1.0` no longer includes `anyio` by default, we'd currently need the following...

```suggestion
    "httpcore",
    "anyio",
```

(Although the more comprehensive approach will be to pull in https://github.com/encode/httpx/pull/2858, which is what we'd like for httpx 1.0)